src/api/setting_engine/mod.rs

```diff
@@ -294,4 +295,11 @@ impl SettingEngine {
     pub fn set_receive_mtu(&mut self, receive_mtu: usize) {
         self.receive_mtu = receive_mtu;
     }
+
+    /// Sets a callback used to generate mid for transceivers create by this side of the RTCPeerconnection.
+    /// By having separate "naming schemes" for mids generated by either side of a connection, it's
+    /// possible to reduce complexity when handling SDP offers/answers clashing.
+    pub fn set_mid_generator(&mut self, f: impl Fn() -> String + Send + Sync + 'static) {
```

Comment:
I think the callback should take an isize argument which is the value of greater_mid value at the time of calling it

Reply:
sure